src/Adapters/BlobAdapterUtility.ts

```diff
@@ -232,3 +237,19 @@ function sendConfigTelemetry(data: I3DScenesConfig) {
         })
     );
 }
+
+/** Helper function to handle all schema migration transformations */
+export function handleMigrations(data: I3DScenesConfig) {
+    addVisualRuleIds(data.configuration.behaviors);
+}
+
+/** Helper function that adds temporary ids to visual rules in behaviors */
+function addVisualRuleIds(data: IBehavior[]) {
+    data.forEach((behavior: IBehavior) => {
+        behavior.visuals.forEach((visual: IVisual) => {
```

Comment:
do we need any null checks? I think all of this should be fine with empty data but we definitely don't want it to bonk on nulls.